### PR TITLE
Add UPGRADING.md

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/level-ws/issues/54

### CODE REVIEW

File: UPGRADING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+You should now do:
+
+```js
+var WriteStream = require('level-ws')
+var db = level('DB')
+var stream = WriteStream(db, { type: 'del' })
+```
+
+`stream.writable` and `stream.readable` states were removed so those properties can no longer be accessed.
```
Comment (vweevers) [inline UPGRADING.md]: I'm not sure that's true, because streams define those too, or at least they used to.

File: UPGRADING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Since encodings was removed from `levelup@2` we decided to remove them from `level-ws` as well.
```
Comment (vweevers) [inline UPGRADING.md]: was -> were

File: UPGRADING.md (hunk 1 above)
Comment (ralphtheninja) [reply UPGRADING.md]: Good catch. `.writable` can still be accessed, whereas `.readable` is undefined.

File: UPGRADING.md [hunk 3]
```diff
@@ -42,6 +42,6 @@ var db = level('DB')
 var stream = WriteStream(db, { type: 'del' })
 ```
 
-`stream.writable` and `stream.readable` states were removed so those properties can no longer be accessed.
+Internal `this.writable` and `this.readable` was removed. However, `this.writable` still exists due to inheritance, but `this.readable` is now `undefined`.
```
Comment (vweevers) [inline UPGRADING.md]: Did you mean "internally"?

Comment (ralphtheninja) [inline UPGRADING.md]: `were`. Fixing.

**Comment by vweevers** [comment]
GH is annoying sometimes, when you comment on the same line at around the same time :) It creates two threads

File: UPGRADING.md (hunk 3 above)
Comment (ralphtheninja) [reply UPGRADING.md]: As in, the properties that used to be defined in _this_ constructor were removed.

**Comment by ralphtheninja** [comment]
Update text on encodings once https://github.com/Level/level-ws/pull/90 has been squashed.

File: CHANGELOG.md [hunk 4]
```diff
@@ -34,7 +34,7 @@
 * Remove `this.{writable,readable}` state (@ralphtheninja)
 * Remove `this._db.isOpen()` checks (@ralphtheninja)
 * Remove patching db from the API (@ralphtheninja)
-* Remove encoding options (@ralphtheninja, @vweevers)
+* Remove default `'utf8'` encoding and per stream encodings (@ralphtheninja, @vweevers)
```
Comment (vweevers) [inline CHANGELOG.md]: why am i listed here, btw?

**Comment by vweevers** [comment]
Should add the removal of `destroySoon` but that's also missing from the changelog, can wait. We can do another pass on the upgrade guide and changelog before release.

**Pull request merged**